makefile

```diff
@@ -0,0 +1,30 @@
+# Default target
+default: show_makefile
+
+# Job to print the content of the Makefile
+show_makefile:
```

Comment:
ah, interesting, I do this sometimes too.  However, in this case this is mostly repeated by the default job.  What I've found useful is 
```bash
default:
    @cat makefile | grep -E ".*:\s+#"
cat_makefile:          # this just cats the makefile
    @cat makefile
```
This usually means by default if just type `make` I only see the lines with a job and a comment.